components/resourceDrawer/index.test.jsx

```diff
@@ -83,12 +83,16 @@ describe("ResourceDrawer view", () => {
       fetch,
     });
     SessionProvider = mockSessionContextProvider(session);
-    accessControl = mockAccessControl();
+    accessControlData = {
+      accessControl: mockAccessControl(),
+      isAcp: true,
+      isWac: false,
```

Comment:
Another instance where we reduce our argument count down to the one `accessControlType`.